src/vs/workbench/contrib/debug/browser/debugActions.ts

```diff
@@ -330,7 +331,7 @@ export class AddWatchExpressionAction extends AbstractDebugAction {
 	static readonly LABEL = nls.localize('addWatchExpression', "Add Expression");
 
 	constructor(id: string, label: string, @IDebugService debugService: IDebugService, @IKeybindingService keybindingService: IKeybindingService) {
-		super(id, label, 'debug-action codicon-add', debugService, keybindingService);
+		super(id, label, 'debug-action ' + icons.watchExpressionsAdd.classNames, debugService, keybindingService);
```

Comment:
We previously had just one add icon which was the plus `add`, now it would be possible for somebody to customise to have multiple different icons for different adds. Does not really matter I guess

Reply:
It's up to you. If it is essential that the same icon is used, we can have a single add icon. Makes sings simpler, for sure

Reply:
Yeah I would go with a single add icon.